configs/pikamon.yml

```diff
@@ -1,14 +1,15 @@
 pikamon:
-  logging:
-    level: debug
   bot:
     maximum-spawn-duration: 600s
```

Comment:
looks like you didn't change this back to 300s
https://github.com/Jac0bDeal/pikamon/pull/28/commits/b775c783ad21655bdd938e2364a75a06415b4938#diff-1f17e1274c36b7b81a8ac61e6d1dfc559c1ad43f31482bbacfc5af2ca7cccb92L3

Reply:
yeah, i increased it to 600s on the deployed version because 5 minutes felt a tad short.

Reply:
sounds good!